exporter/otlpexporter/config.go

```diff
@@ -24,7 +24,4 @@ type Config struct {
 	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 
 	configgrpc.GRPCClientSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
-
-	// The number of workers that send the gRPC requests.
-	NumWorkers int `mapstructure:"num_workers"`
```

Comment:
Why is this being removed?

Reply:
Because we don't use this anymore. gRPC has its own load balancer and connection pool, so we should try not to implement a dummy version of that :)

Reply:
Doesn't look like this was actually used anywhere anyway unless I am missing something